src/osd/PG.cc

```diff
@@ -1397,6 +1397,131 @@ void PG::calc_replicated_acting(
   }
 }
 
+bool PG::recoverable_and_ge_min_size(const vector<int> &want) const
```

Comment:
`recoverable_and_get_min_size`?

Reply:
@Liuchang0812 that implies greater than equal to.

Reply:
It is very hard to understand what the function does from this function name~ Could we use an understandable one?

Reply:
The comments within the function try to explain what the function does - check if recoverable. Any suggestions for a better name based on those comments?